packages/shared/src/toast/Toast.js

```diff
@@ -0,0 +1,96 @@
+// @flow strict
+
+import * as React from 'react';
+import { Animated, View } from 'react-native';
+
+import type { TranslationType } from '../../types/Translation';
+import AdaptableBadge from '../AdaptableBadge';
+import StyleSheet from '../PlatformStyleSheet';
+import type { StylePropType } from '../../types/Styles';
+
+type Props = {|
+  +text: TranslationType,
+  +duration: number,
```

Comment:
There should be `+duration?` when you use default props

Reply:
`+duration?` would indicate that `this.props.duration` might be undefined, which it never will be. In case it is not passed as a prop, it will use the default value.